client/src/components/ui/css/LandingPage.css

```diff
@@ -0,0 +1,31 @@
+@import url("https://fonts.googleapis.com/css2?family=Michroma&display=swap");
+
+html,
+body {
+  height: 100%;
```

Comment:
This might break the rest of the app, since CSS is global by default. I highly recommend that you use ChakraUI to do these styles. It's very easy to pick up since you can pass `height` or `h` as a prop to a component to style it's height. You can use autocomplete most of the time to find the right prop for the property you want to set.

For example, your component can have a `Box` with a `minH` or `minHeight` of `"100vh"`.